README.md

```diff
@@ -31,4 +31,6 @@ Samples
 
 * **[WearStandaloneGoogleSignIn](WearStandaloneGoogleSignIn)** (Java) - Demonstrates using Google sign-in to authenticate user. [Guide](https://developer.android.com/training/wearables/apps/auth-wear)
 
+* **[WearTilesKotlin](WearTilesKotlin)** (Kotlin) - Demonstrates tiles using the new androidX alpha library. [Guide](https://developer.android.com/training/articles/wear-tiles)
```

Comment:
Super small nitpick, let's use the `AndroidX` capitalization:

```suggestion
* **[WearTilesKotlin](WearTilesKotlin)** (Kotlin) - Demonstrates tiles using the new AndroidX alpha library. [Guide](https://developer.android.com/training/articles/wear-tiles)
```

Reply:
👍🏻